apps/lobe-chat/1.36.46/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.36.46
+    image: lobehub/lobe-chat:v1.37.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is one difference in this update that could potentially cause an issue:

@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.36.46
+    image: lobehub/lobe-chat:v1.37.1

The update changes the version of the `lobe-chat` service from version 1.36.46 to 1.37.1. This change would require ensuring compatibility between these two versions, especially since v1.37.1 may include updates, new features, or fixes not present in v1.36.46.

If you plan to use a newer version, please ensure it meets your application's requirements and doesn't introduce unforeseen bugs. You might also want to review any documentation on changes or known issues with the updated version.